Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -265,3 +265,68 @@
   - type: Clothing
     sprite: Clothing/Eyes/Glasses/ninjavisor.rsi
   - type: FlashImmunity
+
+- type: entity
+  parent: [ClothingEyesBase, ShowMedicalIcons]
+  id: ClothingEyesGlassesMedical
+  name: medical glasses
+  description: sunglasses that provide flash immunity and a vital monitoring of organic person.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/medglasses.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Glasses/medglasses.rsi
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Tag
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesMedSec
+  name: medsec glasses
+  description: Upgraded sunglasses that provide flash immunity and also both medical scan and background check of the people around the user.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/medsecglasses.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Glasses/medsecglasses.rsi
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Tag
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesCR
+  name: specialize NT glasses
```

Comment:
```suggestion
  name: commander glasses
```

Reply:
To be fair this one is for all centcom agent but eh il rephrase